tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -83,14 +83,16 @@ Status XlaGpuDeviceFactory::CreateDevices(
   }
   string allowed_gpus =
       session_options.config.gpu_options().visible_device_list();
+  auto parsed_gpus=ParseVisibleDeviceList(allowed_gpus).ValueOrDie();
```

Comment:
Please format properly (or use clang-format)

Reply:
Done. Though clang-format-5.0 formats this file differently that the clang-format in CI. I had to manually format some places.